importexport/guitarpro/importgtp-gp5.cpp

```diff
@@ -1278,7 +1278,7 @@ bool GuitarPro5::readNoteEffects(Note* note)
 			      case 9: fret += 28; break;
 			      default: fret += octave * 12;
 			      }
-		      harmonicNote->setString(note->string());
+		    harmonicNote->setString(note->string());
```

Comment:
Looks like whitespace damage (and yes, this file doesn't really follow the code style of MuseScore)